docs/docs/validation-action.mdx

```diff
@@ -0,0 +1,105 @@
+---
+id: validation-action
+sidebar_label: ValidationAction Class
```

Comment:
There's only one final point to address: `decide how much of the slot validation action rasa docs should move to the action server docs` - here my suggestion is to actually label this page `Slot Validation Actions`, then I can complete it with another section on the implementation of `FormValidationAction` and add a link to rasa docs to learn more about how to use these actions in a bot project. WDYT? @melindaloubser1

Reply:
That sounds like a good split! Anything sdk-implementation level belongs in action server docs  👍

Reply:
Done - ready for review now 😁